include/deal.II/fe/fe_nothing.h

```diff
@@ -83,15 +134,16 @@ class FE_Nothing : public FiniteElement<dim, spacedim>
    *
    * The first argument specifies the reference-cell type.
    *
-   * The second argument denotes the number of components to give this
-   * finite element (default = 1).
+   * Constructor. The second argument, `n_components`, denotes the number of
```

Comment:
```suggestion
   * The second argument, `n_components`, denotes the number of
```
Line 133 already mentions that this is a constructor.